src/modules/m_sslinfo.cpp

```diff
@@ -373,8 +373,24 @@ class ModuleSSLInfo
 		{
 			OperInfo* ifo = i->second;
 			std::string fp = ifo->oper_block->getString("fingerprint");
-			if (MatchFP(cert, fp) && ifo->oper_block->getBool("autologin"))
-				user->Oper(ifo);
+			if (MatchFP(cert, fp))
```

Comment:
You could do an early continue here if `!MatchFP` to avoid excessive indentation.

Reply:
Yes, that seems like a good idea.